CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## 1.0.0 (March 8, 2017)
+
+- Switched amalgam8 sidecar to use Envoy proxy from Lyft (https://github.com/lyft/envoy)
```

Comment:
I'm for `amalgam8` but the docs seem to capitalize it as `Amalgam8`. Should be consistent (but I would NOT go over all docs and change the letter case...)